### PR TITLE
Fixes #19: Implementing reading time feature and tester file + update package.json

### LINKED ISSUE #19: Implement a reading time feature for the blog post

### PR DESCRIPTION
**Pull request by cyh0968**
I added basic features for text analysis. In the beginning, my goal was to add reading-time features. However, I realized that word count, the level of writing (readability based on Flesch-Kincaid grade level) and reading-time are highly connected.

For instance, the formula for readability is formula_image
To get the level of writing, we need the number of words, and the formula to get reading time is the total number of words / word per minute. Besides the reading-level library that I use to get Flesch_Kincaid Grade, it also returns the count of words.

Since the three basic features are highly coupled, I think that coupling them together as one class will be better than implementing three different functions. I tried to implement the OOP concept into the function. Because an ES6 class does not fully support encapsulation, I used ES5 syntax with the closures concept in JavaScript to implement private members.

The analyzeText() function return analysis object that contains wordCount, readability, readingTime. The analyzeText() function contains two different public query functions, getAnalysis(), and getAsyAnalsis(). In case of using promise channing, getAsyAnalsis() can be used.

The tester file provides queries to check if functions work well, but it will need more functionality. Plus, I updated the package.json file since it caused an error on the previous pull-request 

I'm not familiar with JavaScript so if you have any opinions, feel free to share your opinions. I will refect the opinions.

### CODE REVIEW

File: test/basic_analysis.test.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+}
+
+console.log(analyzeText);
+
+// Test 1: Test Promise Chain Try && Catch
+test('Test 1-1: Test Promise Chain(then)', () => {
+  analyzeText(text1).getAsyAnalysis()
+    .then((data) => console.log(data))
+    .catch((err) => console.log(err));
+});
```
[the remark this answers is not in the document]
Comment (cyh0968) [reply test/basic_analysis.test.js]: which means we are allowed to use ES7 Syntax in our code? Only test files or other source files?

[the remark this answers is not in the document]
Comment (humphd) [reply test/basic_analysis.test.js]: `async/await` are fine in src and tests, yes.

[the remark this answers is not in the document]
Comment (cyh0968) [reply test/basic_analysis.test.js]: I added the async/await test unit.

**Comment by humphd** [comment]
Merged in https://github.com/Seneca-CDOT/telescope/commit/680b8a999c016e5c9389afe25446f4c5cdece1e9

**Pull request closed without merging**